textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -1,13 +1,12 @@
 package edu.uci.ics.textdb.dataflow.nlpextrator;
```

Comment:
I think you mistakenly include code for adding sentiment analysis to the current Nlp. Since sentiment analysis already become a new operator, can you please remove the changes for this file?